docs/content/en/docs/tasks/cluster/cluster-upgrades/vsphere-and-cloudstack-upgrades.md

```diff
@@ -108,7 +115,7 @@ spec:
       ...
 ```
 
->**_NOTE:_** If you have a custom machine image for your nodes you may also need to update your `vsphereMachineConfig` with a new `template`.
+>**_NOTE:_** If you have a custom machine image for your nodes you may also need to update your `vsphereMachineConfig` with a new `template`. Refer to [vSphere Artifacts]({{< relref "../../../reference/artifacts/#vsphere-artifacts" >}}) to build a new OVA template.
 
 and then you will run the command
```

Comment:
```suggestion
and then you will run one of the following commands, depending on whether it is a management or workload cluster:
```

Reply:
I know it's outside of the PR, but I thought it might be nice to clarify that they need to run one of the two following commands.